deploy/terraform/terraform-units/modules/sap_system/app_tier/outputs.tf

```
@@ -1,11 +1,58 @@
-output "nics-scs" {
+output "nics_scs" {
   value = azurerm_network_interface.scs
 }
 
-output "nics-app" {
+output "nics_app" {
   value = azurerm_network_interface.app
 }
 
-output "nics-web" {
+output "nics_web" {
   value = azurerm_network_interface.web
 }
+
+output "nics_scs_admin" {
+  value = azurerm_network_interface.scs_admin
+}
+
+output "nics_app_admin" {
+  value = azurerm_network_interface.app_admin
+}
+
+output "nics_web_admin" {
+  value = azurerm_network_interface.web_admin
+}
+output "app_ip" {
+  value = azurerm_network_interface.app.*.ip_configuration.private_ip_address
+}
+
+output "app_admin_ip" {
+  value = azurerm_network_interface.app-admin.*.ip_configuration.private_ip_address
```

```suggestion
  value = azurerm_network_interface.app_admin.*.ip_configuration.private_ip_address
```
There are other occurrence, please check log [here](https://dev.azure.com/azuresaphana/Azure-SAP-HANA/_build/results?buildId=13528&view=logs&jobId=ed898bca-1be4-5573-ffb5-764df6a725fe&j=ed898bca-1be4-5573-ffb5-764df6a725fe&t=3515a1e6-b72c-5f3d-99f5-7289c6342fae).